src/main/java/TiTi/task/Deadline.java

```diff
@@ -0,0 +1,41 @@
+package TiTi.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents the Deadline task.
+ * Contain String description of the task.
+ * Contain String description of the date of task.
+ * Contain LocalDate object of the date of task (if applicable).
+ * Contain boolean value of whether the task has been completed.
+ */
+public class Deadline extends Task {
+
+    protected String by;
+    protected LocalDate date;
+
+    /**
+     * Constructor for Deadline class.
```

Comment:
Not sure if nouns are allowed. If not, perhaps can put `Constructs Deadline class.` ?